adapters/pubmatic/pubmatic.go

```diff
@@ -640,7 +641,8 @@ func getBidType(bidExt *pubmaticBidExt) openrtb_ext.BidType {
 // Builder builds a new instance of the Pubmatic adapter for the given bidder with the given config.
 func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server config.Server) (adapters.Bidder, error) {
 	bidder := &PubmaticAdapter{
-		URI: config.Endpoint,
+		URI:        config.Endpoint,
+		bidderName: string(bidderName),
```

Comment:
@pm-nilesh-chate , can you add some description about how `bidderName` is planned to be used

Reply:
This is for distinguishing between bidder and adapter name within the Fledge auction object, where hardcore aliases are used for PubMatic bidder which is part of PBS-Core. These changes related to PBS-Core appear straightforward. If you believe adding a comment would be beneficial, I'll include it in the next upcoming PR, which is also related to this.